packages/vanilla/src/components/footer/__stories__/footer.stories.js

```diff
@@ -0,0 +1,28 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, select } from '@storybook/addon-knobs';
+import footer from '../footer.template';
+import '../../../../../styles/scss/components/footer/index.scss';
+// import readme from '../README.md';
+
+storiesOf('footer', module)
+  .addDecorator(withKnobs)
+  // .addParameters({
+  //   readme: {
+  //     sidebar: readme
+  //   },
+  // })
+  .add('Default', () => {
+    const footerTypeOptions = {
+      tall: '',
+      short: 'short',
+    };
+    setTimeout(() => {
+      console.log('DOMContentLoaded');
```

Comment:
@andysherman2121 can you remove the console log here

Reply:
👍